data/blog/how-we-improved-performance-score-accuracy.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'How we improved Performance Score accuracy'
+date: '2024-01-17'
+tags: ['javascript', 'performance', 'web']
+draft: true
+summary: "We're making updates to how Performance Scores are calculated in the Web Vitals module, which will bring them closer to what your users experience."
+images: ['/images/how-we-improved-performance-score-accuracy/hero.jpg']
+layout: PostLayout
+canonicalUrl: performance-score-improvements
+authors: ['edwardgou']
+---
+
+Last year in November, we released the Web Vitals module for Sentry's Performance product. Aside from helping users monitor and improve their Web Vitals, this new module also introduced [Sentry Performance Scores](https://docs.sentry.io/product/performance/web-vitals/#performance-score) for all web browser applications. As a quick reminder, Performance Scores are used to condense multiple Web Vitals and their respective thresholds into an overall score from 0-100 that rates an app's perceived performance, based on real user data for Web Vitals.
+
+Today, we are making updates to how Performance Scores are calculated. These updates will improve the accuracy of these scores by bringing them closer to what users experience on your web apps. To understand how these changes will impact your Performance Scores, it's useful to understand how they used to be calculated. Here's a brief overview:
+
+1. Your users visit your webpage which generate pageload transactions. These transactions contain [Web Vitals](https://docs.sentry.io/product/performance/web-vitals/) (LCP, FCP, FID, TTFB, CLS) and get sent to Sentry.
+2. In the Sentry Web Vitals module, each web vital is aggregated to get the p75 value.
+3. The p75 value of each Web Vital is then put through a non-linear function which generates individual Web Vital component scores. Each component score is then statically weighted and summed to produce an overall Performance Score out of 100 for your web app. You can find out more about how these are calculate [here](https://docs.sentry.io/product/performance/web-vitals/#performance-score).
```

Comment:
```suggestion
3. The p75 value of each Web Vital is then put through a non-linear function which generates individual Web Vital component scores. Each component score is then statically weighted and summed to produce an overall Performance Score out of 100 for your web app. You can find out more about how these are calculate [in our documentation](https://docs.sentry.io/product/performance/web-vitals/#performance-score).
```

It's usually better (for readability, SEO, and A11y) to have more descriptive link titles

Reply:
Also, if the engineering blog supports equation rendering, it would be highly baller and useful to put the whole equation here. People will definitely want to know that without having to click out!

Reply:
Updated! Good suggestion on the equation rendering, I've added some of that to the post!
![image](https://github.com/getsentry/sentry.engineering/assets/83961295/7c705f45-6b63-4c73-9008-dfdfcaa52abc)